src/installer/pkg/projects/netstandard/pkg/Directory.Build.props

```diff
@@ -3,14 +3,14 @@
     <IsFrameworkPackage>true</IsFrameworkPackage>
     <ShortFrameworkName>netstandard</ShortFrameworkName>
     <ProductBrandPrefix>Microsoft .NET Standard</ProductBrandPrefix>
-
-    <ProductBandVersion>2.1</ProductBandVersion>
-    <ProductionVersion>$(ProductBandVersion).0</ProductionVersion>
```

Comment:
Is this property not needed anymore? Was '0' used to specify patch version?

Reply:
It's needed, but it's calculated in the following target, so this is a minor refactor to improve clarity:

https://github.com/dotnet/runtime/blob/9a8f52c0d4ca2e866642444397e17ee297b9089c/src/installer/Directory.Build.targets#L29-L46

Yeah, `.0` here was just the old, hard-to-find way the patch version was defined. Putting it in `eng/Version.props` makes it easier to rev versions to service the repo.